docs/AboutUs.md

```diff
@@ -5,19 +5,19 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `cs2103t_w10_2[at]comp.nus.edu.sg`
```

Comment:
This email doesnt seem to be working. is there a valid email we can change it to?

Reply:
I think it is best we leave it out since there is no valid email we can change to.